cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -72,6 +74,37 @@ public void updateUser(final MetricUser user) {
     }
 
     @Override
+    public void addCustomAttributesToUser(final String userId, final Map<String, Object> customAttributes) {
+        try {
+            final User intercomUser = User.find(userId);
```

Comment:
Nothing much , a exception is thrown and we log a messsage. I've saw that cheddar wraps all intercom logic within try / catch blocks.